### ISSUE TITLE
Make sesame a home app so we can just use it as a launcher

### ISSUE THREAD

**Issue by devxpy** [BranchMetrics/sesame_issues#202]
Hey there, just purchased sesame. 

Truly an amazing piece of software, thanks for making it.

The only thing I really wish it did is if it was a home app so I can Just use sesame as the default launcher.

You can do this in like 2 lines of configuration, just have to modify the AndroidManifest.xml!

```xml
<activity android:name="Home"
            ...
            android:launchMode="singleInstance"
            android:stateNotNeeded="true">
    <intent-filter>
            <action android:name="android.intent.action.MAIN" />
            <category android:name="android.intent.category.HOME"/>
            <category android:name="android.intent.category.DEFAULT" />
    </intent-filter>
</activity>
```

[See SO question1.](https://stackoverflow.com/questions/8248213/use-my-own-android-app-apk-as-launcher-home-screen-replacement)

I've personally shipped apps using this feature, and it works as advertised.

**Comment by HunterArmstrong** [comment]
I like this.

**Comment by boehs** [comment]
I also like this

**Comment by Martin5001** [comment]
I personally wouldn't need this feature because Sesame already provides integration with some 3rd party launchers like Nova and Niagara Launcher.

**Comment by HunterArmstrong** [comment]
> I personally wouldn't need this feature because Sesame already provides integration with some 3rd party launchers like Nova and Niagara Launcher.

@Martin5001 This is true. But, just having sesame as a home screen in general would be a lot simpler for users than just switching to a different launcher.

**Comment by Martin5001** [comment]
> This is true. But, just having sesame as a home screen in general would be a lot simpler for users than just switching to a different launcher.

If I understand you correctly it is about making Sesame act as a launcher. But it's not possible to have an app as "home" (launcher) and still keep the original launcher – basically you'd have to have 2 launchers at once, not possible. So you'd still be "switching to a different launcher" it just would be Sesame instead of your original launcher.

**Comment by HunterArmstrong** [comment]
> > This is true. But, just having sesame as a home screen in general would be a lot simpler for users than just switching to a different launcher.
> 
> If I understand you correctly it is about making Sesame act as a launcher. But it's not possible to have an app as "home" (launcher) and still keep the original launcher – basically you'd have to have 2 launchers at once, not possible. So you'd still be "switching to a different launcher" it just would be Sesame instead of your original launcher.

What he was saying was to have sesame as the default launcher. Not have two at the same time.

**Comment by Martin5001** [comment]
> > > This is true. But, just having sesame as a home screen in general would be a lot simpler for users than just switching to a different launcher.
> > 
> > 
> > If I understand you correctly it is about making Sesame act as a launcher. But it's not possible to have an app as "home" (launcher) and still keep the original launcher – basically you'd have to have 2 launchers at once, not possible. So you'd still be "switching to a different launcher" it just would be Sesame instead of your original launcher.
> 
> What he was saying was to have sesame as the default launcher. Not have two at the same time.

Oh, now I get it, thanks. (I just haven't thought of a launcher without an app drawer, only search. Interesting idea.)

**Comment by HunterArmstrong** [comment]
@Martin5001 Yeah. Np. Ik you would get it.

**Comment by Kordian-Dziwisz** [comment]
I think that's a good idea, and maybe possible, although I'm not an android app dev, I haven't  seen this before, but what about sesame as an overlay instead of fullscreen app like in #137
